section1/reflection.md

```diff
@@ -1,19 +1,30 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I feel that I'm okay asking questions in class (possibly too quickly), and then I tend not to ask questions much when I'm working alone.
 
 2. How would you print the string `"Hello World!"` to the terminal?
+* p "Hello World!" then run the ruby doc from the terminal.
 
 3. What character is used to indicate comments in a ruby file?
+* # (octothorpe)
 
 4. Explain the difference between an integer and a float?
+* Integer will always represent a whole number (1, 2, 3 humans), where as float numbers will use the decimal points (1.33 kilograms)
 
 5. In the space below, create a variable `animal` that holds the string `"zebra"`
+animal = "zebra"
 
 6. How would you print the string `"zebra"` using the variable that you created above?
+p "#{animal}"
```

Comment:
string interpolation isn't necessary here since you aren't combining string/variable into one string. we'd prefer just `p animal`

(in this case, `animal` is already a string. but even if it were an integer or another data type, Ruby will convert it to a string when you use a print statement).